lib/stepmod/utils/converters/stepmod_ext_description.rb

```diff
@@ -32,13 +32,14 @@ def convert(node, state = {})
           <<~TEMPLATE
             === #{linkend.last}
```

Comment:
We want this to be put into the Glossarist gem and serialized out too.

Reply:
We want to use glossarist methods here instead of directly parsing xml?

Reply:
We should do: XML => Glossarist => Metanorma AsciiDoc.

Reply:
Discussed with @ronaldtse, decided to left it like this